tests/smoke/hello_test.go

```diff
@@ -63,6 +63,20 @@ func TestHelloBundle(t *testing.T) {
 	assert.Equal(t, "hello", installations[0]["name"], "expected the hello installation to be output by porter list")
 	assert.Equal(t, "test", installations[0]["namespace"], "expected the installation to be in the test namespace")
 
+	// Search by name
+	output, err = test.Porter("list", "hello", "--namespace=*", "--output=json").Output()
```

Comment:
Should this test now use `"--name=hello"`?

Reply:
I have changed the testdata so that it fails if I mess up the the --name flag now.